apiary-metastore-events/apiary-receivers/apiary-receiver-common/src/test/java/com/expediagroup/apiary/extensions/events/receiver/common/event/EventTypeTest.java

```diff
@@ -13,13 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.expediagroup.apiary.extensions.receiver.common.event;
+package com.expediagroup.apiary.extensions.events.receiver.common.event;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.util.HashMap;
 import java.util.Map;
 
+import org.assertj.core.api.Assertions;
```

Comment:
I don't think you need this import as it is already imported statically above. Just remove `Assertions.` from the code.

Reply:
Aaa yes, must have added it by accident